pkg/appmgmt/general/general_test.go

```diff
@@ -96,6 +98,9 @@ func TestGetAppMetadata(t *testing.T) {
 				"queue":                    "root.b",
 				"yunikorn.apache.org/user": "testuser",
 			},
+			Annotations: map[string]string{
+				constants.SchedulingPolicyStyleParam: "gangSchedulingStyle=Hard",
```

Comment:
Please export the parameter name in a constant and use that one. Also use constants for the possible values